test/Microsoft.Azure.SignalR.IntegrationTests/BasicIntegrationTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Azure.SignalR.Tests.Common;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.Azure.SignalR.IntegrationTests.Infrastructure;
+
+namespace Microsoft.Azure.SignalR.IntegrationTests
+{
+
+    public class IntegrationTestSkeleton : VerifiableLoggedTest
+    {
+        private ITestOutputHelper _output;
+
+        public IntegrationTestSkeleton(ITestOutputHelper output) : base(output)
+        {
+            _output = output;
+        }
+
+        [Fact]
+        public async Task MockServiceConnectionHandshake()
+        {
+            Console.WriteLine();
+            var host = new WebHostBuilder()
+               .ConfigureServices(services =>{})
+               .ConfigureLogging(logging => logging.AddXunit(_output))
+               .UseStartup<IntegrationTestStartup>()
+               .UseUrls("http://localhost:8901")
```

Comment:
Perhaps we should not start a web server ourselves here and find a simpler way to use SDK.

Reply:
can [TestServer](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.testhost.testserver?view=aspnetcore-3.0) be leveraged?

Reply:
👍 thanks for the idea!